docs/tutorials/mobile/helpers/make-dynamic-shape-fixed.md

```diff
@@ -0,0 +1,85 @@
+---
+title: Make dynamic input shape fixed
+descriptions: 
+parent: Helpers
+grand_parent: Mobile
+nav_order: 2
+---
+
+# Making dynamic input shapes fixed
+{: .no_toc }
+
+If a model can potentially be used with NNAPI or CoreML as reported by the [model usability checker](./model-usability-checker.md), it may require the input shapes to be made 'fixed'. This is because NNAPI and CoreML do not support dynamic input shapes. 
+
+For example, often models have a dynamic batch size so that training is more efficient. In mobile scenarios the batch generally has a size of 1. Making the batch size dimension 'fixed' by setting it to 1 may allow NNAPI and CoreML to run of the model.
+
+The helper can be used to update specific dimensions, or the entire input shape.
+
+## Contents
+{: .no_toc }
+
+* TOC placeholder
+{:toc}
+
+## Usage
+```
+python -m onnxruntime.tools.make_dynamic_shape_fixed -h
+usage: make_dynamic_shape_fixed.py:make_dynamic_shape_fixed_helper [-h] [--dim_param DIM_PARAM] [--dim_value DIM_VALUE] [--input_name INPUT_NAME] [--input_shape INPUT_SHAPE] input_model output_model
+
+Assign a fixed value to a dim_param or input shape Provide either dim_param and dim_value or input_name and input_shape.
+
+positional arguments:
+  input_model           Provide path to ONNX model to update.
+  output_model          Provide path to write updated ONNX model to.
+
+optional arguments:
+  -h, --help            show this help message and exit
+  --dim_param DIM_PARAM
+                        Symbolic parameter name. Provider dim_value if specified.
```

Comment:
```suggestion
                        Symbolic parameter name. Provide dim_value if specified.
```
but this would require updating the python script too